sqlcapture/capture.go

```diff
@@ -608,7 +634,8 @@ func (c *Capture) backfillStreams(ctx context.Context) error {
 }
 
 func (c *Capture) backfillStream(ctx context.Context, streamID string) error {
-	var streamState = c.State.Streams[streamID]
+	var stateKey = c.Bindings[streamID].StateKey
```

Comment:
It might be cleaner to modify `backfillStreams()` to provide the randomly selected binding as the argument here rather than just the name, in order to avoid indexing into the bindings map here.

Reply:
It seems useful to still have the `streamID` available for logging and getting the `discoveryInfo`, so passing the binding would require accessing that property of the binding a bunch of times or declaring a new variable, which doesn't seem much better. Let me know if I'm missing anything though.